src/components/BadTextInput.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-text-field
+    @input="update"
+    :disabled="disabled"
+    :label="label"
+    :placeholder="placeholder"
+    :id="id"
```

Comment:
Kseniya says this `:` is a shorthand for `v-bind:` - https://vuejs.org/v2/guide/syntax.html#v-bind-Shorthand